game_engine.properties

```diff
@@ -4,15 +4,3 @@
 ## first.
 
 engine_version = 1.9.0.0
-
-## URL where we will download the list of available maps
-map_list_file = https://raw.githubusercontent.com/triplea-game/triplea/master/triplea_maps.yaml?raw=true
-## Example that would use a local file config file, useful for map building or development
-# map_list_file = triplea_maps.yaml
-
-
-## Location where we download latest lobby description file, this tells us where the latest lobby is located
-lobby_properties_file_url = https://raw.githubusercontent.com/triplea-game/triplea/master/lobby_server.yaml?raw=true
```

Comment:
> No need for a props file. If we want to release to update the values in this file, it is just as well the values be in code.

What about the JFX client and serialization proxy feature toggles?  If we want users to be able to try out those features, they're not going to be able to update code to enable them.  Am I misunderstanding the proposal here?